test-utils/src/main/java/org/hibernate/validator/testutil/DescriptorAssert.java

```diff
@@ -54,7 +55,7 @@ public void containsConversion(Class<?> from, Class<?> to) {
 			}
 
 			if ( !foundMatchingConversion ) {
-				fail( String.format( "<%s> does not contain a conversion from <%s> to <%s>.", actual, from, to ) );
+				fail( String.format( Locale.getDefault(), "<%s> does not contain a conversion from <%s> to <%s>.", actual, from, to ) );
```

Comment:
yes, I was waiting for your response on this one (I was just 90% sure if to remove them or not :) ) - I'll remove `jdk-unsafe` and `jdk-non-portable` from the config as well as that commit with changes caused by it.